tests/queries/0_stateless/02808_custom_disk_with_user_defined_name.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+# Tags: no-fasttest, no-parallel
```

Comment:
```suggestion
# Tags: no-fasttest, no-parallel
```
why no parallel?

Reply:
because of part of test which I deleted just before making a pr.. now not needed